pkg/gateway/multiparts/db_tracker.go

```diff
@@ -92,3 +103,39 @@ func (m *dbTracker) Delete(ctx context.Context, uploadID string) error {
 	})
 	return err
 }
+
+func Migrate(d db.Database, writer io.Writer) error {
+	ctx := context.Background()
+	je := json.NewEncoder(writer)
+
+	// Create header
+	err := je.Encode(export.Header{
+		Version:   kv.MigrateVersion + 1,
+		Timestamp: time.Now(),
+	})
+	if err != nil {
+		return err
+	}
+
+	entries := make([]MultipartUpload, 0)
+	err = d.Select(ctx, &entries, "SELECT * FROM gateway_multiparts")
```

Comment:
we don't want to select all the database content at this point - we like to query the data and migrate each part - can even read the information directly into the proto message